### PR TITLE
Added option to disable species cries

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
Had this made for a bit after some people were asking about it, but myself I don't see it being much useful without determining a "default cry". This PR could be used as a point of discussion as well.

## Feature(s) this PR does NOT handle:
Doesn't remove the "cry" page from the pokedexes. 

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

File: src/pokemon.c [hunk 1]
```diff
@@ -6814,8 +6814,8 @@ void HealBoxPokemon(struct BoxPokemon *boxMon)
 u16 GetCryIdBySpecies(u16 species)
 {
     species = SanitizeSpeciesId(species);
-    if (gSpeciesInfo[species].cryId >= CRY_COUNT)
-        return 0;
+    if (gSpeciesInfo[species].cryId >= CRY_COUNT || P_CRIES_ENABLED == FALSE)
```
Comment (Bassoonian) [inline src/pokemon.c:6817]:
```suggestion
    if (P_CRIES_ENABLED == FALSE || gSpeciesInfo[species].cryId >= CRY_COUNT)
```

**Comment by AsparagusEduardo** [comment]
Applied

**Comment by kittenchilly** [comment]
I feel like this option should also remove the cry page on the pokedex, as it will do literally nothing

**Comment by AsparagusEduardo** [comment]
> I feel like this option should also remove the cry page on the pokedex, as it will do literally nothing

~~Cry about it :joy:~~ /jk

**Comment by AsparagusEduardo** [comment]
Updated the description

**Review by Bassoonian — APPROVED** [review]

**Pull request merged**